newIDE/app/src/UI/TagChips.js

```diff
@@ -1,7 +1,6 @@
 // @flow
 import React from 'react';
 import Chip from '../UI/Chip';
-import randomColor from 'randomcolor';
```

Comment:
I guess you can remove the package as well

Reply:
👋

Reply:
👋  we'll see each other soon I'm sure